src/plugins/data/public/resources/base_resource_client.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpSetup } from 'opensearch-dashboards/public';
+
+const BASE_API = 'api/enhancements';
```

Comment:
prolly for feature branch purposes but data plugin doesn't depend on the enhancements plugin so might have to invert it

Reply:
Yeah, I felt bad about this tbh. I'd like to move the query enhancement languages (`sql`, `ppl`, `promql`) autocompletion definitions to the query enhancement to avoid having this coupling in the data plugin. Thoughts?